region/norfair/east/Spiky Platforms Tunnel.json

```diff
@@ -138,6 +138,52 @@
       "flashSuitChecked": true
     },
     {
+      "link": [1, 1],
+      "name": "X-Mode Shinecharge, R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "h_heatProof",
+        "h_RModeCanRefillReserves",
+        {"enemyKill": {"enemies": [["Tripper"]]}},
+        {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
+        {"or": [
+          {"partialRefill": {"type": "ReserveEnergy", "limit": 5}},
+          {"and": [
+            {"or": [
+              "h_lavaProof",
+              "Grapple"
+            ]},
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}},
+            {"or": [
+              {"enemyKill": {"enemies": [["Tripper"]]}},
+              "canBeLucky"
+            ]}
+          ]}
+        ]},
+        "canBeVeryPatient",
+        "canXMode",
+        "h_XModeSpikeHit",
+        "h_shinechargeMaxRunway",
+        "h_heatTriggerRModeSparkInterrupt",
+        {"or": [
```

Comment:
You could ride/kill the opposite tripper and kill it once you are about to leave the room.

Reply:
That's included in the kill logic (as an alternative to `canBeLucky`). With two trippers it's about 75% to get at least one large energy.